packages/internals/src/engine-commands/getVersion.ts

```diff
@@ -1,18 +1,16 @@
-import Debug from '@prisma/debug'
 import type { NodeAPILibraryTypes } from '@prisma/engine-core'
 import { getCliQueryEngineBinaryType } from '@prisma/engines'
 import { BinaryType } from '@prisma/fetch-engine'
 import { isNodeAPISupported } from '@prisma/get-platform'
 import execa from 'execa'
+import * as TE from 'fp-ts/TaskEither'
 
 import { resolveBinary } from '../resolveBinary'
 import { load } from '../utils/load'
 
-const debug = Debug('prisma:getVersion')
-
 const MAX_BUFFER = 1_000_000_000
 
-export async function getVersion(enginePath?: string, binaryName?: BinaryType): Promise<string> {
+export async function getBinaryVersion(enginePath?: string, binaryName?: BinaryType): Promise<string> {
```

Comment:
I'm confused here, maybe you forgot to delete this file?
Since `packages/internals/src/engine-commands/getBinaryVersion.ts` exists now, it's a duplicate?

Reply:
I think this is the result of `git merge main`. Since this file didn't get signaled as a merge conflict, git simply created a duplicate. Thanks for spotting it